utils/read_xml.py

```diff
@@ -0,0 +1,30 @@
+import os
+from utils import constants
+from lxml import etree
+
+
+def manage_credentials_in_maven(path, reset=False):
```

Comment:
Some nitpicks here:

* The name of the file is not accurate to what it does
* This method should have a more descriptive name.
* Add a comment with a description for the method please